contrib/devtools/Dockerfile

```diff
@@ -4,7 +4,7 @@
 
 FROM bufbuild/buf:1.1.0 as BUILDER
 
-FROM golang:1.17-alpine
```

Comment:
this needs to stay 1.17 cause of the go get

Reply:
bump @faddat, if we must use `1.17`, then let's have the version be explicit

Reply:
oh-- sorry guys 

I fixed this locally and I'll fix it here too.  thanks for pointing it out

Reply:
All set.